src/graphql/resolvers/businessInfoResolver.ts

```diff
@@ -6,6 +6,52 @@ import {
 import { getInterpretationProvider, InterpretationConfig } from '../../providers/interpretation_providers';
 import { BusinessInferences, QueryGetBusinessInfoArgs } from '../generatedTypes';
 import { defaultConfig, DataSourceConfiguration } from '../../models/data_models/dataSourceConfiguration';
+import { BusinessInfoInput } from '../../models/data_models/types';
+import { GraphQLClient, gql } from 'graphql-request';
+
+async function fetchFromDB(
+	endpoint: string,
+	businessInfo: BusinessInfoInput
+): Promise<BusinessInferences | undefined> {
+	const query = gql`
+		query getInfo($name: String!, $address: String!) {
+			allBusinesses(where: { name: $name, address: $address }) {
+				businessInferences
+			}
+		}
+	`;
+	const variables = {
+		name: businessInfo.name,
+		address: businessInfo.address,
+	};
+
+	const client = new GraphQLClient(endpoint);
+	return client
+		.request(query, variables)
+		.then((x) => <BusinessInferences>JSON.parse(x.allBusinesses[0].businessInferences))
+		.catch(() => {
+			return undefined;
```

Comment:
might want to log out an error

Reply:
Yea, I initially did. The point of this is to catch errors as well as "empty responses" and logging the errors might get annoying.